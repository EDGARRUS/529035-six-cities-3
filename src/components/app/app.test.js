import React from "react";
import renderer from "react-test-renderer";
import App from "./app.jsx";

const houses = [`Beautiful & luxurious apartment at great location`, `Wood and stone place`, `Wonderful hosue for rich people`];

it(`<App /> Testing`, () => {
  const tree = renderer.create(<App houses={houses} onTitleButtonClick={() => {}}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
