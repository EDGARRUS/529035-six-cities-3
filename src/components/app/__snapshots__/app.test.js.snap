// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Testing 1`] = `
<div>
  <p>
    Главная страница приложения
  </p>
  <div
    className="offers"
  >
    <h1>
      Предложения
    </h1>
    <h3
      onClick={[Function]}
    >
      Beautiful & luxurious apartment at great location
    </h3>
    <h3
      onClick={[Function]}
    >
      Wood and stone place
    </h3>
    <h3
      onClick={[Function]}
    >
      Wonderful hosue for rich people
    </h3>
  </div>
</div>
`;
